src/Resources/contao/config/ide_compat.php

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
Remove that, please. That's not needed if classes are where they belong :)

Reply:
If you move all classes to PSR-0 or PSR-4 style, you don't need it :) Overriding classes is not possible anymore and it was the absolutely **worst** idea ever to introduce such a messy feature that abused namespacing to allow for class overrides. It's not supported in Contao 4+ anymore.

Reply:
I see, gonna update the fork to reflect that then. Ty